src/neptune/objects/neptune_object.py

```diff
@@ -218,6 +221,10 @@ def __init__(
     On Linux it looks like it does not help much but does not break anything either.
     """
 
+    def _async_create_run(self):
+        """placeholder for async run creation"""
```

Comment:
not really it can be seen as temporary dev implementation that will be removed before release. The docstring is there only to make things more clear for other devs